### ISSUE TITLE
Repoman checks fail

### ISSUE THREAD

**Issue by timeraider4u** [gentoo-science/sci#758]
As already mentioned in my latest pull request #757 the repoman check fails because of 
```bash
LICENSE.invalid [fatal]       1
   sci-libs/amdlibm/amdlibm-3.1.ebuild: AMD
```
Seems that this issue has been introduced by commit https://gitweb.gentoo.org/repo/gentoo.git/commit/licenses?id=1510c8bc10b883ec799f0d6e051768f199c1b5ab to the main-tree. 

Please fix it somehow (e.g. add the license file to the science repo or anything) as it is a total show-stopper for all possible contributors!

**Comment by jlec** [comment]
Fixed in 77c8f39f7ce54d37cfa849e05027a1b8667f0d9f

**Comment by timeraider4u** [comment]
Thank you very much for fixing!
